docassemble/ALToolbox/data/questions/display_template_demo.yml

```diff
@@ -26,7 +26,7 @@ subquestion: |
       - The default is to display the template with a scroll bar. This is helpful if your markdown template content is long.
       - If your markdown template has a subject line, it will be displayed automatically. The subject line is not required for this feature.
   1. **collapse** (default=False) 
-      - Change it to True to collapse/show youre template in order to save screen space. Template subject line is required.
+      - Change it to True to collapse/show you're template in order to save screen space. Template subject line is required.
```

Comment:
```suggestion
      - Change it to True to collapse/show your template in order to save screen space. Template subject line is required.
```

Reply:
Fixed in https://github.com/SuffolkLITLab/docassemble-ALToolbox/commit/d75e34a61f6c36101536a31a5d01adeec322d8b4, sorry for merging quick